mockgcp/mockpubsub/topic.go

```diff
@@ -48,35 +48,40 @@ func (s *publisherService) CreateTopic(ctx context.Context, req *pb.Topic) (*pb.
 	if err := s.storage.Create(ctx, fqn, obj); err != nil {
 		return nil, err
 	}
+
 	return obj, nil
 }
 
 func (s *publisherService) populateDefaultsForTopic(obj *pb.Topic) {
-	obj.MessageStoragePolicy = &pb.MessageStoragePolicy{
-		AllowedPersistenceRegions: []string{
-			"asia-east1",
-			"asia-northeast1",
-			"asia-southeast1",
-			"australia-southeast1",
-			"europe-north1",
-			"europe-west1",
-			"europe-west2",
-			"europe-west3",
-			"europe-west4",
-			"southamerica-west1",
-			"us-central1",
-			"us-central2",
-			"us-east1",
-			"us-east4",
-			"us-east5",
-			"us-east7",
-			"us-south1",
-			"us-west1",
-			"us-west2",
-			"us-west3",
-			"us-west4",
-			"us-west8",
-		},
+	// TODO: When _is_ this populated?
```

Comment:
Good question. I don't know,  but at least in PubSubTopic creation. (I verified against real GCP in the mock storage notification #2447)